ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/CommandDeduplicationIT.scala

```diff
@@ -19,15 +19,17 @@ import scala.concurrent.Future
 import scala.concurrent.duration.DurationInt
 import scala.util.{Failure, Success}
 
-final class CommandDeduplicationIT extends LedgerTestSuite {
+class CommandDeduplicationIT(ledgerTimeIntervalMs: Int) extends LedgerTestSuite {
+  private val timeInterval = ledgerTimeIntervalMs / 1000
+  private val deduplicationSeconds = 1 * timeInterval
+  private val deduplicationWindowWait = 2 * timeInterval
 
   test(
-    "CDSimpleDeduplication",
+    "CDSimpleDeduplicationBasic",
```

Comment:
I thought you already renamed these tests.

Reply:
will rebase